compiler/src/dotty/tools/dotc/typer/Typer.scala

```diff
@@ -2655,6 +2655,10 @@ class Typer extends Namer
         tree.tpe <:< wildApprox(pt)
         readaptSimplified(Inliner.inlineCall(tree, pt))
       }
+      else if (tree.symbol.is(Macro, butNot = Inline)) {
+        ctx.error("Scala 2 macro cannot be used in Dotty", tree.sourcePos)
```

Comment:
I would add a link to http://dotty.epfl.ch/docs/reference/dropped-features/macros.html (which also needs to be updated: https://github.com/lampepfl/dotty/issues/5971)

Reply:
Will add